apps/sms/style/attachment.css

```diff
@@ -97,7 +118,7 @@
   position: absolute;
   bottom: 0.5rem;
   right: 0;
-  left: 0;
+  left: 1px;  /* Avoid issues related to being too much on the left */
```

Comment:
Is this requested by designer or for other reason that leaving 1 px margin for file name? Because it seems no text truncate originally.